tests/braille-specs/sr-cyr-g1_harness.yaml

```diff
@@ -0,0 +1,25 @@
+display: unicode-without-blank.dis
+table:
+  language: sr
+  region: sr-cyr
+  grade: 1
+  __assert-match: sr-cyr.tbl # sr-g1.ctb
+flags: {testmode: bothDirections}
```

Comment:
I think it would be nicer to organize the tests as follows:

- A section for tests that work for both tables in both directions. I'm thinking of the punctuation test for instance.
- A section for forward tests. Because we have two tables that should both work the same in forward direction, the forward tests can be applied to both tables at once.
- Two sections of backward tests, one for each table. Because the tables are expected to behave differently in backward direction, it doesn't make sense to apply all tests to both tables. I think we will be able to eliminate some `xfail`s that aren't technically failures because it is the expected behavior of the table.

Reply:
I fully agree with this arrangement. I'm not yet familiar enough with the harness syntax however, I'll try to do this within the next few days.

Reply:
Let me know if you need help with anything.

Reply:
I'm not exactly sure how to refactor the tests. most tests only differ in the xfail flag, whether to test backtranslation or not. Here's my idea in pseudo-yaml:

```yaml
table: sr.tbl
flags: {testmode: bothDirections}
tests:
- all tests except cyrillic letters
flags: {testmode: forward}
tests:
- cyrillic letters
table: sr-Cyrl.tbl
flags: {testmode: bothDirections}
tests:
- all tests except latin letters
flags: {testmode: forward}
tests:
- latin letters
```

I'm not sure how to fill in "all tests except..." without repeating tests.

Reply:
Sorry, missed your comment. I'm having a look now...

Reply:
Done